crypto/keys/keyring.go

```diff
@@ -501,6 +507,24 @@ func lkbToKeyringConfig(name, dir string, buf io.Reader, test bool) keyring.Conf
 	}
 }
 
+func newKWalletBackendKeyringConfig(name, _ string, _ io.Reader) keyring.Config {
+	return keyring.Config{
+		AllowedBackends: []keyring.BackendType{"kwallet"},
+		ServiceName:     "kdewallet",
```

Comment:
Why not use `name` like everything else?

Reply:
If we use `kdewallet` we'll use `kwallet` default namespace (e.g. user's default keyring). App's name goes into `KWalletAppID`, keys are segretated anyway. I needed to pick one, and I thought that for user's convenience it'd be better to use the default one, but I'm open to reconsider